File: 1/1.rb
# Project Euler #1

sumnum = 0
num = 1000

(num).times do |i|
  if i % 3 == 0
    p i.to_s + " is a multiple of 3!"
    sumnum += i
  elsif i % 5  == 0
    p i.to_s + " is a multiple of 5!"
    sumnum += i 
  end
  #p sumnum
  #p i
end

p sumnum

File: 1/1.c
#include <stdio.h>

int main( )
{
  int sumnum = 0;
  int num    = 1000;
  int i;

  for( i=0; i<num; i++ )
  {
    if( i % 3 == 0 || i % 5 == 0 )
      sumnum += i;
  }

  printf( "%d\n", sumnum );

  return 0;
}
